### ISSUE TITLE
normal users(not sudo users) can't see the wiki pages.

### ISSUE THREAD

**Issue by sakuramochi708** [ueckoken/KokenWiki#164]
一般ユーザーがWikiの画面を閲覧することができなくなった。
３日前の更新になにかトラブルがありそうです。

（以下、DeepL直訳）
General users are no longer able to view the Wiki pages.
There seems to be some trouble with the update 3 days ago.

**Comment by Azuki-bar** [comment]
![image](https://user-images.githubusercontent.com/42642269/122217605-8724c300-cee8-11eb-863e-da0d0f611302.png)
```html
<html>
<head>
  <title>We're sorry, but something went wrong (500)</title>
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <style>
  .rails-default-error-page {
    background-color: #EFEFEF;
    color: #2E2F30;
    text-align: center;
    font-family: arial, sans-serif;
    margin: 0;
  }

  .rails-default-error-page div.dialog {
    width: 95%;
    max-width: 33em;
    margin: 4em auto 0;
  }

  .rails-default-error-page div.dialog > div {
    border: 1px solid #CCC;
    border-right-color: #999;
    border-left-color: #999;
    border-bottom-color: #BBB;
    border-top: #B00100 solid 4px;
    border-top-left-radius: 9px;
    border-top-right-radius: 9px;
    background-color: white;
    padding: 7px 12% 0;
    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
  }

  .rails-default-error-page h1 {
    font-size: 100%;
    color: #730E15;
    line-height: 1.5em;
  }

  .rails-default-error-page div.dialog > p {
    margin: 0 0 1em;
    padding: 1em;
    background-color: #F7F7F7;
    border: 1px solid #CCC;
    border-right-color: #999;
    border-left-color: #999;
    border-bottom-color: #999;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
    border-top-color: #DADADA;
    color: #666;
    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
  }
  </style>
</head>

<body class="rails-default-error-page">
  <!-- This file lives in public/500.html -->
  <div class="dialog">
    <div>
      <h1>We're sorry, but something went wrong.</h1>
    </div>
    <p>If you are the application owner check the logs for more information.</p>
  </div>


<div></div></body>
</html>
```

**Comment by otariidae** [comment]
Unfortunetely `something went wrong` has no useful information...

Application log is below: 

```
NoMethodError (undefined method `resolve_column_aliases' for #<ActiveRecord::PredicateBuilder:0x000029f192491350>):
  
app/controllers/pages_controller.rb:76:in `show_page'
```

https://github.com/ueckoken/KokenWiki/blob/b2bf59074c6186b317816e6dcc491786ccc4983a/app/controllers/pages_controller.rb#L76

ref: https://github.com/CanCanCommunity/cancancan/issues/652
fixed at: https://github.com/CanCanCommunity/cancancan/pull/657
released at: [cancancan 3.2.0](https://github.com/CanCanCommunity/cancancan/blob/develop/CHANGELOG.md#320)

This bug comes from updating Rails to 6.1 (dd62bb2e15e08854946fb7bac5374a882497fe8a).
Updating cancancan to 3.2.0 will solve this.
